src/tooltip/tooltip.scss

```diff
@@ -20,6 +20,7 @@
   @include themed {
     background-color: rgba(0, 0, 0, t(iui-opacity-3));
     color: t(iui-color-foreground-accessory);
+    border: 1px solid rgba(t(iui-color-foreground-accessory-rgb), t(iui-opacity-4));
```

Comment:
Can we avoid this change in default themes? Looks really bad in default dark

Reply:
This one I feel pretty strongly that we need to leave in.  The tooltip could appear above an AppUI frame and we have no idea what color might be underneath.
![Screen Shot 2022-03-23 at 2 14 21 PM](https://user-images.githubusercontent.com/849817/159768023-1f4edba2-88dd-4782-863d-61e350fc8c13.png)
 
 I wouldn't say it looks bad either, both GitHub and VSC have a light border on their tooltips.